code-of-conduct.md

```diff
@@ -33,7 +33,19 @@ Participants asked to stop any harassing behavior are expected to comply immedia
 
 ## Scope
 
-This Code of Conduct applies both at the event and in public spaces when an individual is representing Our Networks.
+This Code of Conduct applies at the event, in the online spaces of the conference, and in public spaces when an individual is representing Our Networks.
+
+## Content Moderation Guidelines
+
+Because of ensuring this code of conduct is enforcable in an online context across platforms, we have some guidelines about the types of spaces the conference includes.
```

Comment:
```suggestion
Because of ensuring this code of conduct is enforceable in an online context across platforms, we have some guidelines about the types of spaces the conference includes.
```